ui/dashboards/src/components/GridLayout/GridTitle.tsx

```diff
@@ -58,10 +58,14 @@ export function GridTitle(props: GridTitleProps) {
         backgroundColor: ({ palette }) =>
           palette.mode === 'dark' ? palette.background.paper : palette.background.default,
       }}
+      data-testid="panel-group-header"
     >
       {collapse ? (
         <>
-          <IconButton onClick={collapse.onToggleOpen}>
+          <IconButton
+            onClick={collapse.onToggleOpen}
+            aria-label={`${collapse.isOpen ? 'collapse' : 'expand'} group ${title}`}
```

Comment:
I did fix this one a11y thing because it was very simple and low risk.

Reply:
I have some aria-label stuff coming in upcoming PR (mostly just moving around aria-label stuff that is already there) -- if you like the approach we could standardize (but it might be over-engineering)